Amend Documents (#10)
estie-inc/cder: README.md

cder (_see-der_) is a database seeding tool to help you import fixure data in your local environment.

Generating seeds programmatically is an easy task, but maintaining them is not.

> Everytime when your schema is changed, your seed can be broken.

Reviewer: ```suggestion
Every time when your schema is changed, your seed can be broken.
```

> It costs your team extra efforts to keep them updated.

Reviewer: ```suggestion
It costs your team extra effort to keep them updated.
```

Author: I fixed as suggested, but may I ask for an explanation in detail for future reference, if you would not mind?
For clarity, just let me tell what I thought: `efforts` here referred to humans' 'actions' required to update seeds, that is, rewriting or adding seeds data and keeping them new, and therefore `[countable] an attempt to do something especially when it is difficult to do` [1] sounded suited to me. 
[1] https://www.oxfordlearnersdictionaries.com/definition/english/effort

Reviewer 2: That makes sense to me (actually my initial intention).
Having said that, it's not a critical point in terms of usage of this crate, so I'd accept either way (please correct if you will!) Thanks again for the detailed review 👍

#### with cder you can:
- maintain your data in a readable format, separated from the seeding program
- handle reference integrities on-the-fly, using **embedded tags**

> - reuse existing structs and insert function, with only a little glue code is needed

Reviewer: ```suggestion
- reuse existing structs and insert function,s with only a little glue code is needed
```

Reviewer 2: @kenkoooo  perhaps you meant `s` comes into before comma, right?

```suggestion
- reuse existing structs and insert functions, with only a little glue code is needed
```

Reviewer: Thanks! Good catch!

  email: ${{ ENV(DEVELOPER_EMAIL:-"developer@example.com") }}
```

> Without specifying the defalut value, all the tags that point to undefined environment vars are simply replaced by empty string "".

Reviewer: ```suggestion
Without specifying the default value, all the tags that point to undefined environment vars are simply replaced by empty string "".
```

### Data representation
cder deserializes yaml data based on [serde-yaml](https://github.com/dtolnay/serde-yaml), that supports powerful [serde serialization framework](https://serde.rs/). With serde, you can deserialize pretty much any struct. You can see a few [sample structs](tests/test_utils/types.rs) with various types of attributes and [the yaml files](tests/fixtures) that can be used as their seeds.

> Shown below are a few basics of required YAML format.

Reviewer: ```suggestion
Below are a few basics of required YAML format.
```

Author: Okay, let us keep things simple here.
